docs/source/Quickstart/03_psi.md

```diff
@@ -1,37 +1,44 @@
 # Create a PSI instance
 
 ## Steps
+
 1. **Download openrc.sh**
 
 ```
 source openrc.sh
 ```
 
 2. **Create VM**
-Find a flavor satisfying preferred requirements
 
-```
-openstack flavor list -c Name -c RAM -c Disk -c VCPUs -f value | awk '$2>64 && $3>100 && $4>12'
+Find a flavor satisfying [preferred requirements](https://ci-framework.readthedocs.io/en/latest/Quickstart/01_requirements.html)
```

Comment:
in markdown, we should be able to use `[preferred requirements](./01_requirements.md)` - for instance:
https://stackoverflow.com/questions/60438785/sphinx-and-markdown-hyperlinks-to-section-in-a-different-file.

That should make things easier/better for the future.

Reply:
Indeed better, thanks.